### PR TITLE
Let reader read old (pre-sharded) logs. Needed so latest reader (in d…

### PR DESCRIPTION
**Pull request by MikeStall**
…ashboard) can still consume older writers.

### CODE REVIEW

File: src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    // LogReader needs to handle older format.
+    // DO this in one central place so that we can remove this class when deprecate the old format.
```
Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs]: Let's create a bug to remove this, after the old versions are removed from the feed post GA

File: src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            }
+            return null;
+        }
+
+        // -1 if not matched
+        public static long GetEpochFromTable(CloudTable table)
+        {
+            if (table.Name == OldTableName)
+            {
+                return 201609;
```
Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs]: What does this mean?

File: src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    // LogReader needs to handle older format.
+    // DO this in one central place so that we can remove this class when deprecate the old format. 
+    // One central place 
+    internal class LegacyHack
```
Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs]: nit: I'd rename this to LegacyTableHelper - if this ever were to surface in a stack trace, that name would make people feel better. If we create a bug as I suggested above to remove this, we'll remember w/o the name :)

File: src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs (hunk 1 above)
Comment (MikeStall) [inline src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs]: Filed: https://github.com/Azure/azure-webjobs-sdk/issues/876

File: src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs (hunk 2 above)
Comment (MikeStall) [inline src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs]: September 2016 - suffix number for before version .7; used to keep things in order.

**Comment by mathewc** [comment]
:ship:

File: src/Dashboard/app/views/AboutHome.html [hunk 4]
```diff
@@ -3,4 +3,5 @@
         <h3>About</h3>
         <ng-include src="'app/views/IndexerLogTable.html'"></ng-include>
     </div>
-</div>
\ No newline at end of file
+</div>
+1.2
```
Comment (mathewc) [inline src/Dashboard/app/views/AboutHome.html]: what is this?

**Pull request merged**